src/docs/product/session-replay/getting-started.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: "Set Up"
+sidebar_order: 6
+description: "Get started with Sentry's Session Replay, which allows you to visualize a video-like reproduction of the user's session."
+---
+
```

Comment:
Removed `<Include name="beta-note-session-replay.mdx" />` from this docs since this include also lists all platforms.

All links from here include it, so we should be good.